### PR TITLE
Update reviewers list

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by foolip** [comment]
For this to work the new reviewers need to be invited to the org. I'll do that.

File: gamepad/META.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 spec: https://w3c.github.io/gamepad/
 suggested_reviewers:
   - marcoscaceres
+  - jameshollyer
```
Comment (foolip) [inline gamepad/META.yml]:
```suggestion
  - JamesHollyer
```

**Review by foolip — APPROVED** [review]
Both invited, I suggest waiting for them to confirm that they've accepted before merging this.

**Comment by gsnedders** [comment]
@sagoston still hasn't accepted the invite to the org.

**Comment by foolip** [comment]
@marcoscaceres can you revert that line and we'll land the other?

**Review by foolip — APPROVED** [review]

**Pull request merged**